README.md

```diff
@@ -16,6 +18,11 @@ Castle Core is &copy; 2004-2017 Castle Project. It is free software, and may be
 
 ## Building
 
+| Platform | Build Status |
+|----------|-------------:|
+| Windows  | [![Build status](https://ci.appveyor.com/api/projects/status/49a6i0ydiku56r5b/branch/master?svg=true)](https://ci.appveyor.com/project/castleproject/core) |
```

Comment:
This should probably link to just master builds too:
https://ci.appveyor.com/project/castleproject/core/branch/master